packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/SyncedRealmTests.kt

```diff
@@ -1238,44 +1238,6 @@ class SyncedRealmTests {
         }
     }
 
-    @Test
```

Comment:
We have seen errors that we messed up the logging so that users only received log messages if they set the correct log levels at the right moment, but maybe that was an issue with the previous two competing concepts (RealmLog vs. LogConfiguration) 🤔 ... I guess it should be consistent and safe now

Reply:
Yes, before there were some conditions were log levels were overwritten after setting a config due to this LogConfiguration vs RealmLog.

It is not an issue now, sync logs are received depending on the category level.